kin-ecosystem-sdk/src/main/java/com/kin/ecosystem/network/api/OrdersApi.java

```diff
@@ -513,6 +515,12 @@ public Call getHistoryCall(String X_REQUEST_ID, Integer limit, String before, St
 
         List<Pair> localVarQueryParams = new ArrayList<Pair>();
         List<Pair> localVarCollectionQueryParams = new ArrayList<Pair>();
+        if (limit != null) {
```

Comment:
why limit != null should it be origin!=null? also below offerId!=null

Reply:
👍